content/en/docs/cluster-administration/cluster-wide-alerting-and-notification/notification-manager.md

```diff
@@ -0,0 +1,450 @@
+---
+title: "Manage multi-tenant notifications with Notification Manager"
+keywords: "kubernetes, KubeSphere, notification manager, email, wechat, slack"
+description: "K8s native notification management with multi-tenancy support"
+
+linkTitle: "Notification Manager"
+weight: 2020
+---
+
+[Notification Manager](https://github.com/kubesphere/notification-manager) manages notifications in KubeSphere. It receives alerts or notifications from different senders and then send notifications to different users.
+
+Supported senders includes:
+
+- Prometheus Alertmanager
+- Custom sender (Coming soon)
+
+Supported receivers includes:
+
+- Email
+- [Wechat Work](https://work.weixin.qq.com/)
+- Slack
+- Webhook (Coming soon)
+
+![Notification Manager](/images/docs/cluster-administration/cluster-settings/cluster-wide-alerting-and-notification/notification-manager.png)
+
+## QuickStart
+
+### Config Prometheus Alertmanager to send alerts to Notification Manager
+
+Notification Manager uses port `19093` and API path `/api/v2/alerts` to receive alerts sending from Prometheus Alertmanager of Kubesphere.
+
+To receive Alertmanager alerts, **KubeSphere already added Alertmanager webhook and route configurations like below** ( by editing the Secret alertmanager-main in the namespace `kubesphere-monitoring-system` ):
+
+Send Prometheus alerts to Notification Manager:
+
+```yaml
+    "receivers":
+    - "name": "prometheus"
+      "webhook_configs":
+      - "url": "http://notification-manager-svc.kubesphere-monitoring-system.svc:19093/api/v2/alerts"
+    "route":
+      "routes":
+      - "match":
+          "alerttype": ""
+        "receiver": "prometheus"
+```
+
+Send event alerts to Notification Manager:
+
+```yaml
+    "receivers":
+    - "name": "event"
+      "webhook_configs":
+      - "url": "http://notification-manager-svc.kubesphere-monitoring-system.svc:19093/api/v2/alerts"
+        "send_resolved": false
+    "route":
+      "routes":
+      - "match":
+          "alerttype": "event"
+        "receiver": "event"
+        "group_interval": "30s"
+```
+
+Send auditing alerts to Notification Manager:
+
+```yaml
+    "receivers":
+    - "name": "auditing"
+      "webhook_configs":
+      - "url": "http://notification-manager-svc.kubesphere-monitoring-system.svc:19093/api/v2/alerts"
+        "send_resolved": false
+    "route":
+      "routes":
+      - "match":
+          "alerttype": "auditing"
+        "receiver": "auditing"
+        "group_interval": "30s"
+```
+
+{{< notice note >}}
+
+The above is the default configuration. If you do not want to receive a certain type of alert, you can delete the corresponding configuration.
+
+{{</ notice >}}
+
+### Config receivers
+
+Notification Manager now supports three type receivers, email, Wechat Work, slack. Only administrator can config receivers.
```

Comment:
Notification Manager now supports three type receivers, email, Wechat Work, slack. Only administrator can config receivers.
->
Notification Manager now supports three types of receivers: Email, WeChat Work and Slack. Only the administrator can configure receivers.